cfdm/data/netcdfarray.py

```diff
@@ -123,6 +95,15 @@ def __init__(
 
                 .. versionadded:: (cfdm) 1.10.0.0
 
+            ncvar:  Deprecated at version 1.10.1.0
+                Use the *addresss* parameter instead.
```

Comment:
So it doesn't sound like a snake wrote this!

```suggestion
                Use the *address* parameter instead.
```

Reply:
:)